src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -270,7 +270,16 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
 
   async checkAudioConnectivity(audioInputDevice: Device): Promise<CheckAudioConnectivityFeedback> {
     let audioPresence = false;
+    const checkAudioConnectivityMetrics = {
+      audioPacketsSent: 0,
+      audioPacketsReceived: 0,
+    };
     const audioVideo = this.meetingSession.audioVideo;
+
+    const checkAudioConnectivityMetricsObserver: AudioVideoObserver = this.getCheckAudioConnectivityMetricsObserver(
```

Comment:
Probably inline the observer here itself like the `checkVideoConnectivity` approach over a class method? 

If we are making it a class level method then would be better to make it re-usable so that `metricsDidReceive` is attached once and populates both audio and video metrics and each check relies on that. That would be out of scope for this though.

Reply:
Will create inline observer for now.